index.html

```diff
@@ -1,111 +1,86 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"
+    rel="stylesheet" />
 
-          <li><img src="./image/Icon - Medium.png" alt="Medium" /></li>
+  <title>Mobile Porfolio</title>
+</head>
 
-          <li><img src="./image/Github.png" alt="github" /></li>
-          <li><img src="./image/angelist 1.png" alt="n" /></li>
-        </ul>
-      </div>
-    </section>
-    <section class="work">
-       <div class="project tonic-project">
-          <img
-            src="image\Snapshoot Portfolio11.png"
-            alt="tonic"
-            class="project-img"
-          />
+<body>
+  <header class="mobile-portfolio">
+    <a href="#" class="logo">
+      <h2>Eros</h2>
+    </a>
+    <nav class="menu"><img id="open-menu" src="./image/Union.svg" alt="union" />
+      <ul class="navigation d-none" id="navigation">
+        <!-- <li class="item-navigation"> -->
+        <li><img id="cross" class="cross-menu" src="./image/cross.png" alt="cross-menu"></li>
+        <li> <a class="menu-name" href="#work">Portfolio</a></li>
+        <li><a class="menu-name" href="#about">About</a></li>
+        <li><a class="menu-name" href="#form">Contact</a></li>
+
+      </ul>
+    </nav>
+  </header>
+
+  <section class="first-page" id="first-page">
+    <h1 class="name">
+      I'm Aime <br />
+      Glad to see <br> you!
+    </h1>
+
+    <p class="text">
+      I am a software developer! I can help you build a product, feature or
+      website Look through some of my work and experience! If you like what
+      you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <div class="contact">
+      <a href="#" class="connect">LET'S CONNECT</a>
+      <ul class="social-icons">
+        <li><a href="#"><img src="./image/Twitter.png" alt="twitter" /></a></li>
+        <li> <a href="#"><img src="./image/Icon - Linkedin.png" alt="dot"></a></li>
 
-        <div class="project-details">
+        <li> <a href="#"><img src="./image/Icon - Medium.png" alt="Medium" /></a></li>
+
+        <li> <a href="#"><img src="./image/Github.png" alt="github" /></a></li>
+        <li> <a href="#"><img src="./image/angelist 1.png" alt="n" /></a></li>
+      </ul>
+    </div>
+  </section>
+
+  <section class="work" id="work">
+    <!--
+    <div class="project tonic-project">
+       <img
+          src="./image/Snapshoot Portfolio.svg"
+          alt="tonic"
+          class="project-img"
+        />
+
+      <div class="project-details">
+        <div class="details">
           <h2 class="project-title">Tonic</h2>
           <nav class="project-nav">
             <h4 class="project-client">Canopy</h4>
+
             <img src="./image/Counter.svg" alt="counter" />
```

Comment:
* Kindly remove all  commented codes from your codebase as it makes your code bulky and unprofessional